seed_data/management/commands/create_tiers_test.py

```diff
@@ -59,3 +59,11 @@ def test_create_tier_with_add_to_existing_parameter(self):
         self.command.handle("create_tiers", tiers=4, program_id=None, add_to_existing=True)
         assert TierProgram.objects.filter(program=self.program1).count() == 4  # Default is 4
         assert TierProgram.objects.filter(program=self.program2).count() == 4
+
+    def test_zero_threshold_and_discount(self):
+        """
+        Test that there is one TierProgram with income_threshold=0 and one with discount_amount=0
+        """
+        self.command.handle("create_tiers", tiers=4, program_id=None, add_to_existing=False)
```

Comment:
so here we are making sure that there are 2 entries one with `income_threshold=0` and other with `discount_amount=0`? 

The code says that there can be one entry with both `income_threshold=0` and `discount_amount=0`.